pkg/helpers/legalnotice/legalnotice.go

```diff
@@ -0,0 +1,42 @@
+package legalnotice
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/klog"
+)
+
+// Display the necessary legal notice if it's configured in the system.
+// Note that the legal notice comes from the "legal-notice" configMap, which
+// should be in the "openshift-config" namespace. This needs to be configured
+// by the deployer.
+func DisplayLegalNotice(coreClient corev1client.CoreV1Interface, out io.Writer) error {
+	legalnoticecm, err := coreClient.ConfigMaps("openshift-config").Get("legal-notice", metav1.GetOptions{})
+	if kerrors.IsNotFound(err) || kerrors.IsForbidden(err) {
+		// NOTE(jaosorior): We shouldn't get here given that a clusterRole and
```

Comment:
Not quite sure what you're talking here about? Just say it's fine if the CM does not exists. Forbidden though, is something unexpected, imo.

Reply:
I had an idea initially... but I don't think it's relevant anymore. So I'll remove the forbidden check.